database.sql

```diff
@@ -492,6 +492,24 @@ CREATE TABLE IF NOT EXISTS `cache` (
 	 INDEX `k_expires` (`k`,`expires`)
 ) DEFAULT COLLATE utf8mb4_general_ci COMMENT='Stores temporary data';
 
+--
+-- TABLE channel
+--
+CREATE TABLE IF NOT EXISTS `channel` (
+	`id` int unsigned NOT NULL auto_increment COMMENT '',
+	`uid` mediumint unsigned NOT NULL COMMENT 'User id',
+	`label` varchar(64) NOT NULL COMMENT 'Channel label',
+	`description` varchar(64) COMMENT 'Channel description',
+	`access-key` varchar(1) COMMENT 'Access key',
+	`include-tags` varchar(255) COMMENT 'Comma separated list of tags that will be included in the channel',
+	`exclude-tags` varchar(255) COMMENT 'Comma separated list of tags that aren\'t allowed in the channel',
+	`full-text-search` varchar(255) COMMENT 'Full text search pattern, see https://mariadb.com/kb/en/full-text-index-overview/#in-boolean-mode',
+	`media-type` smallint unsigned COMMENT 'Filtered media types',
```

Comment:
Please add a `created` and `modified` column just to be safe.

Reply:
Hmm, what is the benefit? It is just some user setup.

Reply:
I can't think about a specific use for these at the moment but I've lived long enough that when we need that data it better be there.